packages/application-shell-connectors/src/components/application-context/normalizers.ts

```diff
@@ -98,23 +98,14 @@ type NormalizedGroupedByResourceType = {
   // E.g. { orders: {...} }
   [key: string]: NormalizedGroupedByPermission | null;
 };
-/**
- * dataFence: {
- *   store: {
- *     orders: {
- *       canManageOrders: { values: ['usa', 'germany'] },
- *       canViewOrders: { values: ['canada'] },
- *     }
- *   }
- * }
- */
 type NormalizedStoresGroupByResourceType = {
```

Comment:
Not subject matter in this PR but this often confuses me. what normalized implies here.

Reply:
I would not call this normalized either. it's the raw datafence we get from the API

Reply:
we can follow this up some time.
adding it to my backlog

Reply:
> it's the raw datafence we get from the API

Not really, from the API we get a list of data fences

```
type StoreDataFence implements DataFence {
  value: String!
  name: String!
  group: String!
  type: String!
}
union AppliedDataFence = StoreDataFence

allAppliedDataFences: [AppliedDataFence!]!
```

And here we need to normalize the list into a nested object.

I don't know how else you want to call this 🤔

Reply:
Thanks. Makes sense. Forgot about this union to allow potentially future types of data fences.